extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.js

```diff
@@ -66,7 +67,16 @@ export default class StaticWadoClient extends api.DICOMwebClient {
       actual = actual.Alphabetic;
     }
     if (typeof (actual) == 'string') {
-      return actual.indexOf(desired) != -1;
+      if (actual.length === 0) return true;
```

Comment:
Compare with wild cards - not strictly needed for the e2e tests, but I prefer to keep things close to original.